pages/mobile/base.py

```diff
@@ -64,6 +64,19 @@ def wait_notification_box_visible(self):
     def wait_notification_box_not_visible(self):
         self.wait_for_element_not_visible(*self._notification_locator)
 
+    def login(self, email, password):
+        from fxapom.pages.desktop.sign_in import SignIn
+        fxa_login = SignIn(self.testsetup)
+        fxa_login.sign_in(email, password)
+
+    def create_new_user(self):
+        acct = FxATestAccount(use_prod=False).create_account()
+        return MockUser(email=acct.email, password=acct.password, name=acct.email.split('@')[0])
+
+    def get_default_user(self, user='default'):
+        acct = self.testsetup.credentials[user]
+        return MockUser(email=acct['email'], password=acct['password'], name=acct['email'].split('@')[0])
+
```

Comment:
Actually, these look pretty much identical to the same named methods in `pages/desktop/consumer_pages/base.py`. I would consider creating a higher level object that both `base.py`s could inherit from, which would avoid this duplication.